freestyle-logging/jvm/src/main/scala/loggingJVM.scala

```diff
@@ -50,4 +52,39 @@ object loggingJVM {
           ME.catchNonFatal(logger.warn(msg, cause))
       }
   }
+
+  implicit def freeStyleLoggingKleisli[M[_], C: Manifest](
+      implicit ME: MonadError[M, Throwable]): FSHandler[LoggingM.Op, Kleisli[M, Logger, ?]] =
+    λ[FunctionK[LoggingM.Op, Kleisli[M, Logger, ?]]](op =>
+      Kleisli { logger =>
+        ME.catchNonFatal {
+          op match {
+            case LoggingM.DebugOP(msg) =>
+              logger.debug(msg)
+            case LoggingM.DebugWithCauseOP(msg, cause) =>
+              logger.debug(msg, cause)
+            case LoggingM.ErrorOP(msg) =>
+              logger.error(msg)
+            case LoggingM.ErrorWithCauseOP(msg, cause) =>
+              logger.error(msg, cause)
+            case LoggingM.InfoOP(msg) =>
+              logger.info(msg)
+            case LoggingM.InfoWithCauseOP(msg, cause) =>
+              logger.info(msg, cause)
+            case LoggingM.WarnOP(msg) =>
+              logger.warn(msg)
+            case LoggingM.WarnWithCauseOP(msg, cause) =>
+              logger.warn(msg, cause)
+          }
+        }
+    })
+
+  implicit def freeStyleLoggingKleisliRunner[M[_]](
+      log: Logger): FSHandler[Kleisli[M, Logger, ?], M] =
+    λ[FunctionK[Kleisli[M, Logger, ?], M]](_.run(log))
+
+  implicit def freeStyleLoggingToM[M[_]: MonadError[?[_], Throwable]](
+      log: Logger): FSHandler[LoggingM.Op, M] =
```

Comment:
I think making `log` implicit here will allow both Kleisli style injection as well as implicit style for max flexibility. People wanting to not use implicit DI can interpret to Kleisli and defer injection to the runtime and those providing implicit injection will automatically pick up this handler if for example they are interpreting to `Future` but have an implicit instance. We can move this forward if this `log` value is implicit and you rebase with #352 here. Thanks and great stuff!